service/azureconfig/v1/key/key.go

```diff
@@ -10,6 +10,10 @@ import (
 const (
 	defaultAzureCloudType = "AZUREPUBLICCLOUD"
 
+	clusterTagName      = "GiantSwarmCluster"
+	installationTagName = "GiantSwarmInstallation"
+	organizationTagName = "GiantSwarmOrganization"
```

Comment:
These are the tag names. We can't use the `giantswarm.io/cluster` format we use in AWS because `/` is a restricted character.

> Tag names can't contain these characters: <, >, %, &, \, ?, /

Reply:
Yes, that's sad :(
I like the choice you made given the restrictions.